src/runtime_init_memory.js

```diff
@@ -0,0 +1,82 @@
+#if USE_PTHREADS
```

Comment:
Perhaps a comment at the very top to say roughly what this file is for?   And that its not used in STANDALONE mode?